src/Turbo/src/Twig/TurboStreamListenRendererInterface.php

```diff
@@ -23,5 +23,5 @@ interface TurboStreamListenRendererInterface
     /**
      * @param string|object $topic
      */
-    public function renderTurboStreamListen(Environment $env, $topic): string;
+    public function renderTurboStreamListen(Environment $env, $topic /* , array $eventSourceOptions = [] */): string;
```

Comment:
Regarding the interface and the BC promise

This is a good way to expand an interface while not making current of previous code break. 

But then it also means some class can implement TurboStreamListenRendererInterface without accepting this new parameter.

If you pass a third arrgument to such class, it will break. 

This is why that kind of changes cannot be done (not my opinion here, it's symfony BC promise: https://symfony.com/doc/current/contributing/code/bc.html#changing-interfaces )

Before calling renderTurboStreamListen() on an object implementing TurboStreamListenRendererInterface,  you  need to check if this object is ready to receive this third parameter. Or the best way to do this is to have a dedicated interface. 

That means we probably need to
* create a new interface
* make the mercure streamlistener implement this interface
* type check this on the twig extension/runtime before passing the argument